model/fs_page.php

```diff
@@ -72,6 +73,7 @@ public function __construct($p=FALSE)
          
          $this->show_on_menu = $this->str2bool($p['show_on_menu']);
          $this->important = $this->str2bool($p['important']);
+         $this->orden = isset($p['orden'])?$this->intval($p['orden']):1;
```

Comment:
$this->orden = $this->intval($p['orden']);

No hace falta comprobar si no está definida para ponerla a 1. En el xml ya tienes default a 1.
Me imagino que te falló al hacer el cambio porque no limpiaste la caché.